scrapers/bellapass.yml

```
@@ -0,0 +1,49 @@
+name: bellapass
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - alexismonroe.com
+      - avadawn.com
+      - bellanextdoor.com
+      - bellahd.com
+      - bellapass.com
+      - bryci.com
+      - calicarter.com
+      - janafox.com
+      - joeperv.com
+      - hd19.com
+      - hunterleigh.com
+      - katiebanks.com
+      - monroelee.com
+      - taliashepard.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="videoDetails clear"]/h3
+      Details: //div[@class="videoDetails clear"]/p
+      Performers:
+        Name: //li[@class="update_models"]//a
+      Tags:
+        Name: //div[@class="featuring clear"][2]//a
+      Image:
+        selector: //base/@href|//img[@class="update_thumb thumbs stdimage"]/@src0_1x
+        concat: "|"
+        postProcess:
+          - replace:
+              - regex: "[|]"
+                with: ""
```

i havent checked all sites yet, from a quick look though some of the sites dont have the `https:` in their base eg bryci, bellanextdoor. While the ui from stash doesnt mind ( you can see the images) you'll get an error if you try to save because the backend expects a valid URL with the schema included.
An extra regex takes care of that
```suggestion
        postProcess:
          - replace:
              - regex: "[|]"
                with: ""
              - regex: "^//"
                with: "https://"
```